CODEOWNERS

```
@@ -1 +1 @@
-* @nlopin @martimalek @div-Leo @jonotrujillo @arturmiglio
+* @martimalek @div-Leo @jonotrujillo @arturmiglio
```

🥹 
Let's remove Leo and add Nelson as well 🙈 😭 
```suggestion
* @martimalek @jonotrujillo @arturmiglio @dornelasnelson  
```